components/chart/src/heatfield/chart-heatfield.ts

```diff
@@ -201,30 +188,44 @@ export class DtChartHeatfield extends _DtHeatfieldMixinBase
   @ViewChild(CdkConnectedOverlay, { static: false })
   _overlay: CdkConnectedOverlay;
 
+  private _destroy$ = new Subject<void>();
+  private _relativeBoundingBox: { top: number; left: number; width: number };
+
+  /**
+   * Highcharts charts object is needed for the xAxis is
+   * set by the chart via the `_initHeatfield` function that
+   * is called by the chart.
+   */
+  private _chartObject: Highcharts.ChartObject;
+
   constructor(
     elementRef: ElementRef,
-    // tslint:disable-next-line: no-forward-ref
-    @Inject(forwardRef(() => DtChart)) @SkipSelf() private _chart: DtChart,
     private _changeDetectorRef: ChangeDetectorRef,
   ) {
     super(elementRef);
-    this._checkChartSupport();
   }
 
   ngOnDestroy(): void {
-    this._destroyed.next();
-    this._destroyed.complete();
+    this._destroy$.next();
```

Comment:
💚 for consistency and actually following our coding standards

Reply:
haha thx :)